l5-operator/controllers/bestie_controller.go

```diff
@@ -195,10 +251,54 @@ func (r *BestieReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *BestieReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	return ctrl.NewControllerManagedBy(mgr).
-		For(&petsv1.Bestie{}).
-		Owns(&appsv1.Deployment{}).
-		Owns(&corev1.Service{}).
-		Owns(&routev1.Route{}).
-		Complete(r)
+	builder := ctrl.NewControllerManagedBy(mgr)
+	builder.For(&petsv1.Bestie{})
+	builder.Owns(&appsv1.Deployment{})
+	builder.Owns(&corev1.Service{})
+	builder.Owns(&networkv1.Ingress{})
```

Comment:
Should we load Ingress even if we are on vanilla k8s? It's not wrong, I'm just thinking that we are only creating ingress if there is no openshift cluster.

Reply:
no harm in it since both vanilla and open-shift have ingress.. was our thinking..